src/Mado/QueryBundle/Services/MetaDataAdapter.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Mado\QueryBundle\Services;
```

Comment:
Can be an object instead of a service this class?

Reply:
Yes, it is more appropriate.